### ISSUE TITLE
Use the terminal to change the working directory

### ISSUE THREAD

**Issue by alemorm** [jupyterlab/jupyterlab#5778]
Hi,

Is it possible to change the working directory displayed by the file browser from the terminal? Meaning when I type the following in the builtin terminal:

`cd /path/`

The file browser reflects that change.

I can see how this would lead to unintended consequences so it could perhaps be added as an extra option in the settings. I noticed the **'Open File from Path...'** option under the File menu so something like a **'Change Path...'** option would be more feasible given the js front-end/kernel separation.

I am dealing with a situation where I have to click though multiple directories in the same session due to the file hierarchy of the servers that I work with. I would also be open to suggestions on how people are dealing with the tediousness of this situation or if there are any existing alternatives.

Also, if there could be a way to open a text file, notebook or any compatible file format directly from the terminal it would be extremely useful!

Other than that, I'm loving jupyterlab so far!

Best,
Alejandro

**Comment by ellisonbg** [comment]
Yeah this would be super helpful. Some design considerations to work through but I think we can figure out a way

**Comment by alemorm** [comment]
Great, thanks for the quick reply! I'm glad it can be of use to people.

**Comment by PEERCRED** [comment]
Related feature: a new terminal should start in the working directory displayed in the browser.

**Comment by dclong** [comment]
Related issues:

https://github.com/jupyterlab/jupyterlab/issues/5056

https://github.com/jupyterlab/jupyterlab/issues/2019
